apps/comps/components/unified-leaderboard/skill-detail/skill-detail-page.tsx

```diff
@@ -52,7 +50,7 @@ export const SkillDetailPage: React.FC<SkillDetailPageProps> = ({
   const { refetch: loadMoreAgents, isLoading: isLoadingMore } = useQuery({
     queryKey: ["load-more-agents", currentOffset],
     queryFn: async () => {
-      const response = await apiClient.getGlobalLeaderboard({
+      const response = await client.leaderboard.getGlobal({
```

Comment:
the AI told me it couldn't use the tanstack client for this because its part of a more complicated query logic than the other cases we've migrated so far.  Please let me know if you disagree and think we should still be using the tanstackClient here (and how)

Reply:
That is surprising here because the getGlobal function is maybe the simplest RPC we have. No args, just call a function and get data back. I wonder if you would have more luck if you used the tanstackClient directly instead of wrapping it in that custom hook?

Reply:
hmm yeah that was actually really easy to update this time, not sure what tripped it up last time.  Anyway switched this one over to the tanstackClient, left the other one below using the lower level client